test/integration/roles/test_s3/tasks/main.yml

```diff
@@ -2,10 +2,10 @@
 # tasks file for test_s3
 # ============================================================
 - name: generate random name for the bucket name
-  shell: echo ansible_test_$RANDOM
+  shell: bash -c 'echo ansible_test_$RANDOM'
```

Comment:
Hey, @michael-dev2rights Thanks for this. While you're here can you change this to use command instead of shell? It shouldn't be shell at all, and was on my to-do list to fix. Thanks. If you're feeling motivated, you could fix the other places shell is used in this file as well. But no obligation, it was something I was going to fix at some point anyway.